lib/pbench/cli/agent/commands/results/__init__.py

```diff
@@ -49,15 +50,9 @@ def move_results(ctx, _user, _prefix, _show_server):
     runs_copied = 0
     failures = 0
 
-    # FIXME: use tempfile
-    temp_dir = os.path.join(
-        config.pbench_tmp,
-        ".".join([os.path.basename(__file__), str(os.getpid())]),
-        controller,
-    )
     try:
-        os.makedirs(temp_dir)
-    except OSError:
+        temp_dir = tempfile.mkdtemp()
```

Comment:
This appears to lose the use of `config.pbench_tmp`.  Perhaps we could just do `tempfile.mkdtemp(dir=config.pbench_tmp, prefix="pbench-move-results.")`?